packages/blade/src/utils/getFocusRingStyles/getFocusRingStyles.web.ts

```diff
@@ -0,0 +1,15 @@
+import type { CSSProperties } from 'styled-components';
+import type { Theme } from '../../components/BladeProvider';
+import { castWebType, makeMotionTime } from '~utils';
+
+function getFocusRingStyles(theme: Theme): CSSProperties {
+  return {
+    outline: `4px solid ${theme.colors.surface.border.primary.muted}`,
+    outlineOffset: '1px',
+    transitionProperty: 'outline-width',
+    transitionDuration: castWebType(makeMotionTime(theme.motion.duration['2xquick'])),
+    transitionTimingFunction: castWebType(theme.motion.easing.standard.effective),
```

Comment:
We have motion for this?

Reply:
This was there only Button motion. I will still confirm